.github/workflows/build-docs.yml

```diff
@@ -18,6 +18,7 @@ jobs:
           python-version: 3.7
       - name: Install dependencies
         run: |
+          sudo apt install optipng
```

Comment:
I'm not completely `sudo` works in github workflows. We'll have to check whether the build works when this branch is merged.